src/styles/app/26-node-note-overlays.css

```
@@ -0,0 +1,487 @@
+	flex: 0 0 auto;
+}
+
+.editorColorClearButton {
+	border-color: var(--border-light);
+	background: color-mix(in srgb, var(--bg-secondary) 82%, var(--bg-primary));
+}
+
+.taskInlineDateBtn {
+	position: absolute;
+	left: calc(
+		var(--editor-readable-content-gutter-compact) +
```

Stylelint (17.9.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)